### PR TITLE
feat : modal & grid

### PR DESCRIPTION
**Pull request by mikesch803**
1. modal
2. 3 items in grid
3. 2 items in grid
4. 1 row 3 col in grid

### CODE REVIEW

**Comment by mikesch803** [comment]
> 1. modal
> 2. 3 items in grid
> 3. 2 items in grid
> 4. 1 row 2 col in grid

File: modal.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+console.log(containerModal)
+console.log(btnModalClose)
+console.log(btnModalOpen)
+var openModelHandler = () => {
+    containerModal.style.display = "flex";
+    btnModalOpen.style.display = "none";
+    console.log(true)
+}
+
+var closeModelHandler = () => {
```
Comment (shahbazibnafzal) [inline modal.js]: use let and const where possible, like this function can be declared with const keyword.

File: index.html [hunk 2]
```diff
@@ -306,9 +306,38 @@ <h3>ratings</h3>
     <span class="iconify rating-fill" data-icon="mdi:star" data-inline="false"></span>
     <span class="iconify rating-fill" data-icon="mdi:star" data-inline="false"></span>
 </div>
-
     <span class="rating-num">5<span class="iconify" data-icon="mdi:star" data-inline="false"></span></span>
 
-</body>
+<!-- modal -->
+<h3>modal</h3>
```
Comment (shahbazibnafzal) [inline index.html:312]: wrapping a particular section in a section tag will give more readability

Comment (mikesch803) [reply index.html:312]: it's component site, will do in doc site

File: modal.js (hunk 1 above)
Comment (mikesch803) [reply modal.js]: fixed

**Review by shahbazibnafzal — APPROVED** [review]
Looks good now

File: modal.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+const containerModal = document.querySelector(".modal");
+const btnModalOpen = document.querySelector(".btn-modal-open");
+const btnModalClose = document.querySelector(".btn-modal-close");
+
+containerModal.style.display = "none";
+
+const openModelHandler = () => {
+    containerModal.style.display = "flex";
+    btnModalOpen.style.display = "none";
+    console.log(true)
```
Comment (vrinda-mahajan) [inline modal.js:10]: You can remove console.log from the file. Everything else looks good.

**Review by vrinda-mahajan — APPROVED** [review]

**Review by foolhardy21 — APPROVED** [review]
Looks good. Instead of giving border to each row and col, you can use gap.

File: modal.js (hunk 3 above)
Comment (mikesch803) [reply modal.js:10]: yeah, i forgot to remove this

**Pull request merged**